targets/UnrealMarketplacePlugin/examplesource/ExampleProject/Plugins/PlayFab/Source/PlayFabCommon/Public/TestFramework/PlayFabTestCase.h.ejs

```diff
@@ -53,39 +45,44 @@ public:
      * Before testing, this function is called to gather the list of tests to run for each TestCase.
      * It is not considered part of any test. A failure or exception in this method will halt the test framework.
      */
-    virtual void GetTests(TArray<UPlayFabTestContext*>& InTestList) { }
+    virtual void GetTests(TArray<UPlayFabTestContext*>& InTestList) {}
+
+    /**
+     * Before testing, if these tests require TesTitleData, they'll be given by this method
+     */
+    virtual void SetTestTitleData(UTestTitleDataLoader& testTitleData) {}
```

Comment:
I can't envision testTitleData ever needing to be modified by a specific test.  Pass by const&.